### DISCUSSION TITLE
OverlayScrollbar Tutorial

### DISCUSSION THREAD

**Discussion by kor1998** [KingSora/OverlayScrollbars#615] [General]
As I find about the tutorial I didn't find any code related to overlay scrollbars in Youtube so can you make a tutorial using javascript or React so we can understand the code and use it in our projects please this is a personal request

**Comment by KingSora** [comment]
@kor1998 Youre absolutely welcome to do any tutorial about overlayscrollbars on any platform! :) 

Please share the link here when its finished, I'm excited to see it as well

**Comment by kor1998** [comment]
I refer to you to make a video so I can learn from you this is what I wanna try to say

**Comment by AgentSmith0** [comment]
Why do need a video tutorial, just look at the examples in the docs: https://kingsora.github.io/OverlayScrollbars/examples

**Comment by kor1998** [comment]
@AgentSmith0 Because I am a beginner I need to know how it works by giving direct code I can't understand how it works so that's why I am referring to him making a video tutorial so everyone can understand and everyone can do it. If you find a tutorial related to this OverlayScrollbars please provide me a YouTube link so I can use it on my personal projects works

**Comment by kor1998** [comment]
@KingSora if you are seeing this so please understand that I need a tutorial from you I want to add it to my projects I don't want to see ugly scrollbars. I added it on the body but it refused to style when I saw it on the console it said It refused to style because of Strice MIME control so I asked you to make a tutorial video so I can understand it step-by-step so I can use it my projects

**Reply by KingSora** [reply]
@kor1998 This error seems to be related to something else and is not specific to `OverlayScrollbars`... 

Regarding the video tutorial: I'm currently too busy to do one and I'm also not a Youtuber or anything.. please understand that I can provide tutorials and examples in code or text form but not in video form. At least thats my current stance on this
